getting-started/templates/Azure/azure_supplemental_values

```diff
@@ -0,0 +1,67 @@
+global:
```

Comment:
Add a comment block to the top that describes what this file is and how it should be used.